ui/app/controllers/vault/cluster/clients/current.js

```diff
@@ -0,0 +1,3 @@
+import Controller from '@ember/controller';
```

Comment:
Since we removed all the `loading` actions from the child route (right?) Can we remove empty controllers for the children?

I _think_ I was getting errors when I had tried doing this because I was calling `controllerFor` in the loading function...but i'm not sure

Reply:
Good call @hellobontempo these should be good to remove now.